src/main/java/com/example/schoolmanagement/model/StudentGetDto.java

```diff
@@ -0,0 +1,73 @@
+package com.example.schoolmanagement.model;
+
+import java.time.LocalDate;
+
+public class StudentGetDto {
```

Comment:
```suggestion
public class StudentGetDto {
```
public class StudentResponseDto {